rai/show.go

```diff
@@ -127,3 +127,42 @@ func (tx *TransactionResult) Show() {
 		}
 	}
 }
+
+func zip(lists ...[]interface{}) func() []interface{} {
+	zip := make([]interface{}, len(lists))
+	i := 0
+	return func() []interface{} {
+		for j := range lists {
+			if i >= len(lists[j]) {
+				return nil
+			}
+			zip[j] = lists[j][i]
+		}
+		i++
+		return zip
+	}
+}
+
+func (tx *TransactionAsyncResult) Show() {
+	res := make(map[string][][]interface{})
+
+	for _, r := range tx.Results {
+		res[r.RelationID] = append(res[r.RelationID], r.Table)
+	}
+
+	for k, v := range res {
+		fmt.Println(k)
+		iter := zip(v...)
+		for tuples := iter(); tuples != nil; tuples = iter() {
+			for i, tuple := range tuples {
+				if i > 0 {
+					fmt.Print(", ")
+				}
+
+				fmt.Print(tuple)
+			}
+			fmt.Println()
+		}
```

Comment:
I guess this outer loop is already iterating the tuples, not a vector of tuples?
```suggestion
		for tuple := iter(); tuple != nil; tuple = iter() {
			for i, element := range tuple {
				if i > 0 {
					fmt.Print(", ")
				}

				fmt.Print(element)
			}
			fmt.Println()
		}
```

Reply:
oh thanks :pray: